### PR TITLE
Upgrade to shopsys framework 7.0.0-beta1

### PR DESCRIPTION
**Pull request by simara-svatopluk**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Upgrade the SSFW
|New feature| Yes
|BC breaks| Yes
|Fixes issues| 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -108,11 +108,11 @@
     "require-dev": {
         "ext-pgsql": "*",
         "ext-zip": "*",
-        "codeception/codeception": "^2.4.0",
+        "codeception/codeception": "2.4.5",
```
Comment (vitek-rostislav) [inline composer.json]: could you please provide a better description of the change? Would not it be better to mark the "bad" codeception version (> 2.4.5 I guess) as conflicting?

Comment (vitek-rostislav) [reply composer.json]: also, it would be the best to make this change in a separate commit

**Review by vitek-rostislav — COMMENTED** [review]
Hi, the changes are ok with me, I have only one thing in `composer.json` - please check it out. Also, I noticed that the build is failing on CI - could you investigate it and fix it, please?

File: composer.json (hunk 1 above)
Comment (simara-svatopluk) [reply composer.json]: I removed it completely and fixed codeception problem in a proper way in the newest commit :tada: (not tested yet)

Comment (simara-svatopluk) [reply composer.json]: I also rebased it all, but since You didn't have more questions I consider it to be fine

**Review by vitek-rostislav — COMMENTED** [review]
Hi, now it is fine with me - I would just appreciate more descriptive commit message of the last commit (codeception upgrade) - please add the info as a bullet to a new line of the commit message, you can use the sentence from your commit message from our monorepo - "allowed populating database before all tests because since version 2.5.0 populating option is also checked during loading dump, see https://github.com/shopsys/shopsys/pull/502)"

**Review by vitek-rostislav — APPROVED** [review]
Hi, everything looks ok now :+1:

**Pull request merged**